### PR TITLE
must-gather: collect csiaddons object

### PR DESCRIPTION
**Pull request by yati1998**
This commit collects the following csiaddons object details in must-gather:
1. csiaddonsnode
2. reclaimspacejob
3. reclaimspacecronjob
4. networkfence

Signed-off-by: yati1998 <ypadia@redhat.com>

### CODE REVIEW

**Comment by yati1998** [comment]
@agarwal-mudit @Madhu-1 @nixpanic

**Review by Madhu-1 — APPROVED** [review]

File: must-gather/collection-scripts/gather_namespaced_resources [hunk 1]
```diff
@@ -163,3 +163,32 @@ if [ "$(oc get storageclassclaim --no-headers -A | awk '{print $2}')" != "" ] ;
...
+fi
+
+echo "collecting dump of oc get reclaimspacecronjobs all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+if [ "$(oc get reclaimspacejob --no-headers -A | awk '{print $1}')" != "" ] ; then
+     { oc get reclaimspacecronjobs --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/get_reclaimspacecronjobs_all_ns"
+     { oc describe reclaimspacecronjobs --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/desc_reclaimspacecronjobs_all_ns"
+     { oc get reclaimspacecronjobs -oyaml --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/get_yaml_reclaimspacecronjobs_all_ns"
+fi
+
+echo "collecting dump of oc get netwrokfence all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
```
Comment (agarwal-mudit) [inline must-gather/collection-scripts/gather_namespaced_resources:189]: nit: netwrokfence == networkfence

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**